misc/prime_seive.c

```diff
@@ -0,0 +1,73 @@
+/**
+ * @file
+ * @brief [Prime Seive](https://leetcode.com/problems/count-primes/)
+ * algorithm implementation.
+ * @author [Divyansh Kushwaha](https://github.com/webdesignbydivyansh)
+ */
+#include <assert.h>	/// for assert
+#include <stdio.h>	/// for standard input output
+#include <stdlib.h>	/// for general purpose standard library
```

Comment:
```suggestion
#include <stdlib.h>	/// for general purpose standard library

```

Reply:
sorry, i could not get what you are trying to change in that sentence...

Reply:
ok..i got it now :)